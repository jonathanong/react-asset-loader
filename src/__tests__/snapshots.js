
import renderer from 'react-test-renderer'
import assert from 'assert'
import React from 'react'

import AssetLoader from '..'

const src = 'https://js.stripe.com/v2/'

it('should load the assets', async () => {
  let component = renderer.create(
    <AssetLoader urls={[src]}>
      <div />
    </AssetLoader>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
  assert(tree.props.assets)
  // assert(tree.props.assets[src])
  // assert(!tree.props.assets[src].fulfilled)
  // assert(!tree.props.assets[src].rejected)
  assert(!tree.props.assetsFulfilled)
  assert(!tree.props.assetsRejected)

  await AssetLoader.AssetMap.get(src)
})
