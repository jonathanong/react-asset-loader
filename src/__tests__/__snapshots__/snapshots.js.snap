// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load the assets 1`] = `
<div
  assets={Object {}}
  assetsFulfilled={false}
  assetsRejected={false}
  urls={
    Array [
      "https://js.stripe.com/v2/",
    ]
  }
/>
`;
